src/sagemaker/processing.py

```diff
@@ -1144,6 +1172,41 @@ def _to_request_dict(self):
         return s3_output_request
 
 
+class RunArgs(object):
```

Comment:
This seems like a good class to just use `attr` to define

```suggestion
class RunArgs(object):
    code = attr.ib()
    inputs = attr.ib(default=None)
    outputs = attr.ib(default=None)
    arguments = attr.ib(default=None)
```